src/app/zap-templates/zcl/data-model/chip/tls-certificate-management-cluster.xml

```diff
@@ -36,7 +36,7 @@ Git: 1.4-534-g3214b3502
   </struct>
 
   <cluster apiMaturity="provisional">
-    <domain name="General"/>
+    <domain name="Cameras"/>
```

Comment:
TLS is outside the Camera domain. So, maybe it's best to leave it as General. Although I am not sure how this domain information is used.

Reply:
Yes, `General` seems better. I think the domain is used by the ZAP UI to group clusters.